benchmark_runner/common/ocp_resources/create_odf.py

```diff
@@ -28,10 +28,20 @@ def create_odf(self):
             if resource.endswith('.sh'):
                 # build sgdisks path dynamically
                 if '01_sgdisks.sh' == resource:
-                    sgdisk_list = ''
-                    for disk_name in self.__ibm_blk_disk_name:
-                        sgdisk_list += f'sgdisk --zap-all /dev/{disk_name};'
-                    self.__oc.run(cmd=f'chmod +x {os.path.join(self.__path, resource)}; {self.__path}/./{resource} "{sgdisk_list}"')
+                    worker0_sgdisk = ''
```

Comment:
Couldn't you do something like

```
worker_sgdisks = []
for node, disk_ids in...:
    m = re.match(r'worker-([[:digit:]]+)
    if m:
        node_idx =int(m.group(1))
        for disk_id in disk_ids:
            worker_sgdisks[node_idx] += f'sgdisk --zap-all /dev/disk/by-id/{disk_id};'
```

I still don't like the idea of hard coding what worker names look like in code.  Perhaps put the worker names in the secrets?